rootfs/etc/s6/mariadb/run

```diff
@@ -29,18 +29,17 @@ then
   echo "DROP DATABASE IF EXISTS test;" >> ${INIT_FILE}
   echo "FLUSH PRIVILEGES;" >> ${INIT_FILE}
 
-  if [[ -n "${MARIADB_USERNAME}" && -n "${MARIADB_DATABASE}" ]]
+  MARIADB_DATABASE=${MARIADB_DATABASE:-$MARIADB_USERNAME}
+  if [ ! -z "${MARIADB_DATABASE}" ]
```

Comment:
I'm using `if [ -n "${MARIADB_DATABASE}" ]` all the time for the dockhippie containers, so please stay on this convention.